ui/cypress/e2e/variables.test.ts

```diff
@@ -139,9 +138,67 @@ describe('Variables', () => {
 
     cy.getByTestID('rename-variable-submit').click()
 
-    cy.get('.cf-resource-name--text').should($s =>
-      expect($s).to.contain('-renamed')
+    cy.getByTestID('notification-success--dismiss').click()
+
+    cy.getByTestID(`variable-card--name Little Variable-renamed`).should(
+      'exist'
```

Comment:
is it possible to find this by text? 
```suggestion
    cy.contains('Little Variable-renamed`)
```
that should cover the `should('exist')` part and doesn't couple the test to the implementation of the test id